src/Snifter/AppOptions.cs

```diff
@@ -19,8 +19,8 @@ public class AppOptions
         public int? FilterProtocol { get; set; }
         public IPAddress FilterSourceAddress { get; set; }
         public IPAddress FilterDestAddress { get; set; }
-        public short? FilterSourcePort { get; set; }
-        public short? FilterDestPort { get; set; }
+        internal UInt16? FilterSourcePort { get; set; }
```

Comment:
Could you use the keyword `ushort?` instead of the .NET type `UInt16?`? Just to keep with the style used elsewhere.

Reply:
Okay, please take a look